data/sql/updates/pending_db_world/rev_1565620827529880771.sql

```diff
@@ -0,0 +1,3 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1565620827529880771');
+
+UPDATE `creature` SET `MovementType` = 1, `spawndist` = 2 WHERE `map` = 658 AND `id` IN (2110,14881);
```

Comment:
This is not correct.

How i can see creatures with 2110 & 14881 are Stranglethorn Tiger & Torn Fin Muckdweller located in Hillsbrad Foothills and Stranglethorn Vale not the npc's from Pit of Saron how you mentioned it. 

In this statement you're updating creatures, not creature_template (Rat creature_template is 2110 and Spider creature_template is 14881).

And also while you testing @pklloveyou , test it properly before commenting.